src/components/SubHeader/EvergreenTaskSubHeader.tsx

```diff
@@ -0,0 +1,106 @@
+import { useQuery } from "@apollo/client";
+import { InlineCode } from "@leafygreen-ui/typography";
+import { usePreferencesAnalytics } from "analytics";
+import Breadcrumbs from "components/Breadcrumbs";
+import Icon from "components/Icon";
+import TaskStatusBadge from "components/TaskStatusBadge";
+import { LogTypes } from "constants/enums";
+import { getEvergreenTaskURL } from "constants/externalURLTemplates";
+import {
+  LogkeeperTaskQuery,
+  LogkeeperTaskQueryVariables,
+  TaskQuery,
+  TaskQueryVariables,
+} from "gql/generated/types";
+import { GET_LOGKEEPER_TASK, GET_TASK } from "gql/queries";
+import { shortenGithash } from "utils/string";
+
+interface Props {
+  buildID: string;
+  execution: number;
+  logType?: LogTypes;
+  taskID: string;
+  testID?: string;
+}
+
+export const EvergreenTaskSubHeader: React.FC<Props> = ({
+  buildID,
+  execution,
+  logType,
+  taskID,
+  testID,
+}) => {
+  const { sendEvent } = usePreferencesAnalytics();
+  const { data, loading } = useQuery<TaskQuery, TaskQueryVariables>(GET_TASK, {
+    variables: { taskId: taskID, execution },
+    skip: logType === LogTypes.RESMOKE_LOGS,
+  });
+  const { data: logkeeperData, loading: logkeeperLoading } = useQuery<
+    LogkeeperTaskQuery,
+    LogkeeperTaskQueryVariables
+  >(GET_LOGKEEPER_TASK, {
+    variables: { buildId: buildID },
+    skip: logType !== LogTypes.RESMOKE_LOGS || !buildID,
```

Comment:
I actually think adding a toast if this query fails is too aggressive since users didn't take any action to invoke this query and wouldn't be able to act on the error. Since we [report GQL errors](https://github.com/evergreen-ci/parsley/blob/5c73d8fb269b1d2f693008553d02ef8903042856/src/gql/GQLProvider.tsx#L13) to Bugsnag I think our bases are covered.